SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -41,13 +40,6 @@ struct FConnectionConfig
 		{
 			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_KCP;
 		}
-#if !(PLATFORM_PS4 || PLATFORM_XBOXONE)
-		// RakNet is not compiled for console platforms.
-		else if (LinkProtocolString == TEXT("RakNet"))
-		{
-			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_RAKNET;
```

Comment:
If we're deprecating RakNet, we should probably inform the user to not use it, rather then use another protocol silently.

Reply:
That's what a release note would do right?